Sources/MediaViewer/UI/PreviewController.swift

```diff
@@ -20,6 +29,11 @@ open class PreviewController: UIViewController {
         options: [.interPageSpacing : UIStackView.spacingUseSystem]
     )
     
+    // override navigation item to route to the page view controller's navigation item
```

Comment:
For what purpose? How do we use it?

Reply:
removed, it was for setting the title on the navigation item, i now set it directly on the page vc.